CORE/world_extensions/core_world.rb

```diff
@@ -32,8 +31,8 @@ def reset_data_target
     end
 
     def cleanup_world
-        @browser.close if @configuration['CLOSE_BROWSER'] == 'true'
-        @ledger.print_all if @configuration['PRINT_LEDGER'] == 'true'
+        @browser.close if @configuration['CLOSE_BROWSER']
```

Comment:
This sort of thing, if there is a predictable mapping between key and action, could be handled through dynamic dispatch